GlyssenEngine/GlyssenEngine.csproj

```diff
@@ -3,6 +3,11 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Platforms>AnyCPU;x64</Platforms>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
```

Comment:
This probably works, but since it's the standard MIT license I'd use `<PackageLicenseExpression>MIT</PackageLicenseExpression>` instead.